tests/state/test_state_api.py

```diff
@@ -76,3 +79,20 @@ def test_request_cancel_without_backend(self, stub_broker, api_client):
         # if there is not cancel backend should raise an error
         assert res.json["error"] == "The default broker doesn't have a cancel backend."
         assert res.status_code == 500
+
+    def test_scheduled_jobs(self, stub_broker, api_client, do_work, scheduler, frozen_datetime):
```

Comment:
`test_scheduled_jobs` and i don't think you need stub_broker it comes with the scheduler.